src/plist.h

```diff
@@ -291,6 +291,46 @@ static inline Obj PopPlist(Obj list)
     return val;
 }
 
+
+/****************************************************************************
+**
+*F  NewEmptyPlist() . . . . . . . . . . . . . . . .  Create a new empty Plist
+*F  NewImmutableEmptyPlist() . . . . . . . Create a new immutable empty Plist
+*/
+static inline Obj NewEmptyPlist(void)
+{
+    return NEW_PLIST(T_PLIST_EMPTY, 0);
+}
+
+static inline Obj NewImmutableEmptyPlist(void)
+{
+    return NEW_PLIST_IMM(T_PLIST_EMPTY, 0);
+}
+
+/****************************************************************************
+**
+*F  NewPlistFromArray(<list>,<length>) . . . . .Create a Plist from a C array
+*/
+static inline Obj NewPlistFromArray(Obj * list, Int length)
+{
+    if(length == 0) {
```

Comment:
```suggestion
    if (length == 0) {
```
Perhaps run `clang-format` to be sure?

Reply:
Bah, I could have sworn I clang-formatted this (but I definitely didn't as indeed this is wrong).

I haven't clang-formatted the whole PR, as it moves lots of the lines which call these new functions, making them look strange in their local context